projects/TD3D/td3d/instance_seg_eval.py

```diff
@@ -27,8 +27,7 @@ def aggregate_predictions(masks, labels, scores, valid_class_ids):
         label = label.clone().numpy()
         score = score.clone().numpy()
         info = dict()
-        n_instances = mask.max() + 1
-        for i in range(n_instances):
+        for i in range(mask.shape[0]):
```

Comment:
Please write one more sentences about our changes in the commit before this function